frontend/apps/reader/modules/readerwikipedia.lua

```diff
@@ -391,13 +391,13 @@ function ReaderWikipedia:lookupWikipedia(word, box, get_fullpage, forced_lang)
     -- Fix lookup message to include lang and set appropriate error texts
     local no_result_text, req_failure_text
     if get_fullpage then
-        self.lookup_msg = T(_("Getting Wikipedia %2 page:\n%1"), "%1", lang:upper())
-        req_failure_text = _("Failed requesting Wikipedia article.")
-        no_result_text = _("There was no such Wikipedia article.")
+        self.lookup_msg = T(_("Retrieving Wikipedia %2 article:\n%1"), "%1", lang:upper())
+        req_failure_text = _("Failed retrieving Wikipedia article.")
+        no_result_text = _("No such Wikipedia article found.")
```

Comment:
`Wikipedia article doesn't exist.`?

Reply:
I'll go with ```Wikipedia article not found.``` (less definitive than _does not exist_ :)